backend/libs/json_multipart_parser.py

```diff
@@ -0,0 +1,23 @@
+
+from django.http import QueryDict
+from rest_framework.parsers import MultiPartParser, DataAndFiles
+from rest_framework.settings import api_settings
+from rest_framework.utils import json
+
+
+class JsonMultiPartParser(MultiPartParser):
+    media_type = 'multipart/form-data'
+    strict = api_settings.STRICT_JSON
```

Comment:
It seems unused, we could remove it?

Reply:
It is used, on line 16.

Reply:
There was an unused import though, I removed it.

Reply:
Yes sorry, my comment was not very precise.

What's the goal of adding this option? It looks like that we need to support only one of the case. If yes, I think we could remove the option.

It's also hard to know what's the default value of this option (it should not be added to the settings/common.py file?).

Reply:
I went to DRF's JSONParser for inspiration and found this option that I wasn't aware of. I included it here as well so that no matter what the project's settings are, the behavior of json and multipart-json would be the same.

Reply:
Thanks for the explanation. Makes sense.